### PR TITLE
Support wavefront obj

### PR DESCRIPTION
**Pull request by aochiai**

### CODE REVIEW

**Comment by aochiai** [comment]
This PR depends on https://github.com/ToyotaResearchInstitute/hsr_meshes/pull/1

File: robots/hsrb4s.urdf.xacro [hunk 1]
```diff
@@ -45,11 +45,15 @@ POSSIBILITY OF SUCH DAMAGE.
     <xacro:include filename="$(find hsr_description)/urdf/wrist_v0/wrist.urdf.xacro" />
     <xacro:include filename="$(find hsr_description)/urdf/hand_v0/hand.urdf.xacro" />
 
+    <!-- This switches a format of mesh files for visual property -->
+    <xacro:arg name="use_obj_for_visual" default="false" />
+    <xacro:property name="g_use_obj_for_visual" value="$(arg use_obj_for_visual)"/>
```
Comment (Kazuto-TRI) [inline robots/hsrb4s.urdf.xacro]: What does this prefix "g_" mean?

Comment (aochiai) [reply robots/hsrb4s.urdf.xacro]: `g_` means the property is defined in global scope.

Comment (Kazuto-TRI) [reply robots/hsrb4s.urdf.xacro]: I got it.

**Comment by Kazuto-TRI** [comment]
LGTM.
I don't think it's the best configuration, but I can't come up with any better one.
So let's try this anyway

**Pull request merged**